Packs/EWS/Integrations/EwsExtensionEXOPowershellV2/EwsExtensionEXOPowershellV2.ps1

```diff
@@ -0,0 +1,402 @@
+        [string]$identity,
+        [int]$limit
+    ) {
+        $cmd_params = @{}
+        if ($identity) {
+            $cmd_params.Identity = $identity
+        }
+        if ($limit -gt 0){
+            return Get-EXORecipientPermission @cmd_params -ResultSize $limit
+        }
+        else
+        {
+            return Get-EXORecipientPermission @cmd_params -ResultSize Unlimited
+        }
+        <#
+        .DESCRIPTION
+        Use the Get-EXORecipientPermission cmdlet to view information about SendAs permissions that are
+        configured for users in a cloud-based organization.
+        This command is available only in the Exchange Online PowerShell V2 module
+
+        .PARAMETER identity
+        The Identity parameter specifies the mailbox you want to view.
+        For the best performance, we recommend using the following values:
+            * User ID or user principal name (UPN)
+            * GUID
+
+        Otherwise, you can use any value that uniquely identifies the mailbox. For example:
+            * Name
+            * Alias
+            * Distinguished name (DN)
+            * Domain\Username
+            * Email address
+            * LegacyExchangeDN
+            * SamAccountName
+        #>
+    }
+        [PSObject]GetEXORecipient(
+        [string]$identity,
+        [int]$limit
+    )
+        {
+            $cmd_params = @{ }
+            if ($identity)
+            {
+                $cmd_params.Identity = $identity
+            }
+            if ($limit -gt 0)
+            {
+                return Get-EXORecipient @cmd_params -ResultSize $limit
+            }
+            else
+            {
+                return Get-EXORecipient @cmd_params -ResultSize Unlimited
+            }
+            <#
+            .DESCRIPTION
+            Use the Get-ExORecipient cmdlet to view existing recipient objects in your organization.
+            This cmdlet returns all mail-enabled objects (for example,
+            mailboxes, mail users, mail contacts, and distribution groups).
+
+            #>
+        }
+}
+
+function GetEXORecipientCommand {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)][ExchangeOnlinePowershellV2Client]$client,
+        [hashtable]$kwargs
+    )
+    $identity = $kwargs.identity
+    $limit = $kwargs.limit -as [int]
+    $raw_response = $client.GetEXORecipient($identity, $limit)
+    $human_readable = TableToMarkdown $raw_response "Results of $command"
+    $entry_context = @{"$script:INTEGRATION_ENTRY_CONTEXT.EXORecipient(obj.Guid === val.Guid)" = $raw_response }
+    return Write-Output $human_readable, $entry_context, $raw_response
+}
+
+function GetEXORecipientPermissionCommand {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)][ExchangeOnlinePowershellV2Client]$client,
+        [hashtable]$kwargs
+    )
+    $identity = $kwargs.identity
+    $limit = $kwargs.limit -as [int]
+    $raw_response = $client.GetEXORecipientPermission($identity, $limit)
```

Comment:
please handle Access Right to parse properly in the human readable output

Reply:
https://github.com/demisto/content/pull/11314